app/_src/production/secure-deployment/manage-control-plane-permissions-on-kubernetes.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Manage control plane permissions on Kubernetes
+content_type: how-to
+---
+
+{% capture docs %}/docs/{{ page.release }}{% endcapture %}
+{% assign Kuma = site.mesh_product_name %}
+{% assign kuma = site.mesh_install_archive_name | default: "kuma" %}
+
+By default, {{ Kuma }} deployed on Kubernetes reacts to events and observes all resources at the cluster scope. This approach benefits first-time users who want to explore its functionality and simplifies migration into the mesh. However, in production environments, restricting access to specific resources can enhance security and ensure that {{ Kuma }} does not impact running applications.
+
+Starting from version 2.11, {{ Kuma }} includes new features that let you limit the permissions it needs in Kubernetes.
+
+## Restrict permissions to selected namespaces
+
+One of the new features allows you to define a list of namespaces that {{ Kuma }}'s control plane can access. When this list is set, {{ Kuma }} will only have permissions in those selected namespaces and in its own system namespace. It won't be able to access or manage resources in any other namespace.
+
+### Set allowed namespaces during installation
+
+To restrict {{ Kuma }} to a specific set of namespaces, set the following option during installation:
+
+{% cpinstall namespaceAllowList %}
+namespaceAllowList={kuma-demo}
+{% endcpinstall %}
+
+Replace `kuma-demo` with a comma-separated list of namespaces you want {{ Kuma }} to manage.
+
+This will create a `RoleBinding` in each listed namespace, binding the `{{ kuma }}-control-plane-write` `ClusterRole` to that namespace. It will also configure {{ Kuma }}'s mutating and validating webhooks to only work within the specified namespaces.
```

Comment:
Wasn't there an issue about renaming these RB? Should someone take it?

Reply:
https://github.com/kumahq/kuma/pull/13586